scripts/biomedical/ICTV_Taxonomy/scripts/create_virus_taxonomic_ranking_enums.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Author: Samantha Piekos
+Date: 02/21/2024
+Name: create_virus_taxonomic_ranking_enums.py
+Description: Creates hierarchical viral taxonomy enum schema from the
+ICTV Virus Metadata Resource.
+@file_input: ICTV Virus Metadata Resource .xslx file
+@file_output: formatted .mcf files for viral taxonomy enum schema
+"""
+
+# load environment
+import pandas as pd
+import sys
+
+
+# declare universal variables
+HEADER = [
+'sort',\
```

Comment:
nit: I don't think you need `\` for lists etc that are delimited by `,`

Reply:
removed from all three python script files